src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-hcp-consul.mdx

```diff
@@ -0,0 +1,172 @@
+      ##  Set environment variables and run the collector
+        * Set the HCP Access Token and Access URL values in the `.env` file
+        * Set the New Relic API key variable to your New Relic license ingest key in the `.env` file
+
+```bash
+  # Open the HCP Consul example directory
+  cd newrelic-opentelemetry-examples/other-examples/collector/hcp-consul
+
+  # Set environment variables.
+
+  # run the collector in docker
+  docker compose up
+```
+
+      ### Local Variable information
+
+        <table>
+          <thead>
+            <tr>
+              <th style={{ width: "200px"}}>
+                Variable
+              </th>
+              <th>
+                Description
+              </th>
+              <th>
+                Docs
+              </th>
+            </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  `NEW_RELIC_API_KEY`
+                </td>
+                <td>
+                  New Relic Ingest API Key
+                </td>
+                <td>
+                  [API Key docs](/docs/apis/intro-apis/new-relic-api-keys/)
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  `NEW_RELIC_OTLP_ENDPOINT`
+                </td>
+                <td>
+                  Default US New Relic OTLP endpoint is https://otlp.nr-data.net:4318
+                </td>
+                <td>
+                  [OTLP endpoint config docs](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app/#review-settings)
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  `HCP_ACCESS_TOKEN`
+                </td>
+                <td>
+                  ACL Admin token for HCP Consul
+                </td>
+                <td>
+                  [Docs for creating an ACL admin token](https://developer.hashicorp.com/hcp/docs/consul/hcp-managed/access#generate-admin-token)
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  `HCP_ACCESS_URL`
+                </td>
+                <td>
+                  URL to access your specific Consul server
+                </td>
+                <td>
+                  [docs to locate your access URL](https://developer.hashicorp.com/hcp/docs/consul/hcp-managed/access#get-access-url)
+                </td>
+              </tr>
+            </tbody>
+        </table>
+
+    </Step>
+    <Step>
+      ## HCP Consul Envoy Metrics
+        To monitor HCP Consul Envoy metrics in New Relic, you can configure the OpenTelemetry collector built into your HCP Consul cluster to forward metrics to your own collector. This can include the collector running in this example.
+
+
+        First you will need to add an http reciever to the OpenTelemetry collector. you can add this into the `receivers` group in the example `collector.yaml` file.
+
```

Comment:
```suggestion
      ## Forward HCP Consul Envoy metrics to New Relic [#forward-envoy-metrics]
      
        To monitor HCP Consul Envoy metrics in New Relic, you can configure the OpenTelemetry collector built into your HCP Consul cluster to forward metrics to your own collector. This can include the collector running in this example.


        1. Add an HTTP receiver to the OpenTelemetry collector. You can add this into the `receivers` group in the example `collector.yaml` file:

```